src/librustc_typeck/diagnostics.rs

```diff
@@ -1223,6 +1223,18 @@ fn main() {
 ```
 "##,
 
+E0090: r##"
+The wrong number of lifetimes were supplied. For example:
```

Comment:
Can you specify it is only when _too few lifetimes_ are supplied that you get this error? When it is too many the error is [E0088](https://doc.rust-lang.org/error-index.html#E0088).